src/components/Display/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ number, changeNumber }) => (
+    <div>
+        <input type="text" value={number} onChange={() => changeNumber(number) } /> 
```

Comment:
The display should be editable? 🤔 
If it is a display I think it should be read only

Reply:
yes, of course